libraries/azure-app-configuration-importer/src/internal/parsers/defaultConfigurationSettingsConverter.ts

```diff
@@ -36,39 +38,64 @@ export class DefaultConfigurationSettingsConverter implements ConfigurationSetti
     let configurationSettings = new Array<
       SetConfigurationSettingParam<string | FeatureFlagValue>
     >();
+
     let featureFlagsConfigSettings = new Array<SetConfigurationSettingParam<FeatureFlagValue>>();
-    let foundFeatureManagement = false;
-    let featureFlags: any = {};
-    let featureManagementKeyWord = "";
-    let enabledForKeyWord = "";
+    let foundMsFmSchema = false;
+    let foundLegacySchema = false;
+    let legacySchemaFeatureManagementKeyWord = "";
+
+    const featureFlagsDict: any = {};
 
     if (options.format == ConfigurationFormat.Properties && !options.skipFeatureFlags) {
       this.checkFeatureManagementExist(config);
     }
 
-    for (let i = 0; i < Constants.FeatureManagementKeyWords.length; i++) {
+    for (let i = 0; i < Constants.FeatureManagementKeyWords.length - 1; i++) {
       if (Constants.FeatureManagementKeyWords[i] in config) {
-        if (foundFeatureManagement) {
+        if (foundLegacySchema) {
           throw new ArgumentError(
-            "Unable to proceed because data contains multiple sections corresponding to Feature Management."
```

Comment:
Seems it's not necessary to call out the keyword name in this error.

Reply:
I was thinking since we allow multiple sections of Feature Management i.e legacy and the new ms schema. I was thinking its better to let them know which keyword has been repeated. What do you think?